dev-packages/cli/README.md

```diff
@@ -270,6 +271,20 @@ The property `theiaPlugins` describes the list of plugins to download, for examp
 }
 ```
 
+## Autogenerated Application
+
+This package can auto-generate application code for both the backend and frontend, as well as webpack configuration files.
+
+When targeting Electron, the `electron-main.js` script will spawn the backend process in a Node.js sub-process, where Electron's API won't be available. To prevent the generated application from forking the backend, you can pass a `--no-cluster` flag, or set an environment variable like `THEIA_ELECTRON_NO_BACKEND_FORK=1`.
```

Comment:
We should be very clear that:
- `--no-cluster` should be never used in production, it runs everything in one process, like express server, file watching, git location and so on
- `THEIA_ELECTRON_NO_BACKEND_FORK` can cause performance issues by running the express server within the electron main process, with a link to https://www.electronjs.org/docs/tutorial/performance#3-blocking-the-main-process

Reply:
Removed the env var.